authors/apps/profiles/tests/test_profile.py

```diff
@@ -1,9 +1,9 @@
+from django.urls import reverse
 
 from rest_framework import status
 
 # local imports
```

Comment:
This comment is of no use now since you're separating your imports with one blank line.